scripts/rollup/bundles.js

```diff
@@ -629,6 +629,21 @@ const bundles = [
     global: 'ReactEventsSwipe',
     externals: ['react'],
   },
+
+  {
```

Comment:
make sure you add a `react-events/tap.js` and a `react-events/npm/tap.js`. You will also need to add a line in `react-events/package.json` to include the new file.